### DISCUSSION TITLE
Busca por Escolas

### DISCUSSION THREAD

**Discussion by Leonardofreua** [BrasilAPI/BrasilAPI#243] [Ideas]
Olá pessoal, primeiramente gostaria de parabenizá-los pela iniciativa, com certeza com esta idéia vai ser possíveis criar coisas incríveis!

Gostaria de sugerir uma API que possibilita a busca por escolas por meio de diversos filtros: http://educacao.dadosabertosbr.com/api

Também queria tirar uma dúvida: as funcionalidades futuras poderam buscar dados de outras APIs, assim como de arquivos (datasets), tal como CSV, correto? Pergunto isso, pois existem muitos datasets no http://www.dados.gov.br/dataset e que poderiam ser úteis.

Valeu!

**Comment by elirweb** [comment]
a ideia é bacana, e para complementar trazer informações úteis da escola.
